executor/executor.go

```diff
@@ -1346,31 +1346,6 @@ func (e *UnionExec) resultPuller(ctx context.Context, childID int) {
 	}
 }
 
-// Next implements the Executor Next interface.
-func (e *UnionExec) Next(ctx context.Context) (Row, error) {
-	if !e.initialized {
-		e.initialize(ctx, false)
-		e.initialized = true
-	}
-	if e.cursor >= len(e.rows) {
-		result, ok := <-e.resultCh
-		if !ok {
-			return nil, nil
-		}
-		if result.err != nil {
-			return nil, errors.Trace(result.err)
-		}
-		if len(result.rows) == 0 {
-			return nil, nil
-		}
-		e.rows = result.rows
```

Comment:
`e.rows` can also be removed.

Reply:
My mistake, I will close this PR and fix it in another PR. Thanks for remind me.